ui/apps/platform/cypress/integration/vulnerabilities/workloadCves/WorkloadCves.helpers.js

```diff
@@ -92,6 +92,71 @@ export function extractNonZeroSeverityFromCount(severityCountText) {
     ];
 }
 
+/**
+ * Selects the first CVE for the table and opens the exception modal
+ * @param {('DEFERRAL' | 'FALSE_POSITIVE')} exceptionType
+ */
+export function selectSingleCveForException(exceptionType) {
+    const menuOption = exceptionType === 'DEFERRAL' ? 'Defer CVE' : 'Mark false positive';
+    const modalSelector =
+        exceptionType === 'DEFERRAL'
+            ? selectors.deferCveModal
+            : selectors.markCveFalsePositiveModal;
+
+    return cy.get(selectors.firstTableRow).then(($row) => {
+        const cveName = $row.find('td[data-label="CVE"]').text();
+        cy.wrap($row).find(selectors.tableRowMenuToggle).click();
+        cy.get(selectors.menuOption(menuOption)).click();
+
+        // TODO - Update this code when modal form is completed
+        cy.get(`${modalSelector}:contains("${cveName}")`);
+        return Promise.resolve(cveName);
+    });
+}
+
+/**
+ * Selects the first CVE on each of two pages for the table and opens the exception modal
+ * @param {('DEFERRAL' | 'FALSE_POSITIVE')} exceptionType
+ */
+export function selectMultipleCvesForException(exceptionType) {
+    const menuOption = exceptionType === 'DEFERRAL' ? 'Defer CVEs' : 'Mark as false positives';
+    const modalSelector =
+        exceptionType === 'DEFERRAL'
+            ? selectors.deferCveModal
+            : selectors.markCveFalsePositiveModal;
+
+    const cveNames = [];
+
+    // Select the first CVE on the first page and the first CVE on the second page
```

Comment:
Just curious the test can assume at least 2 pages of workload CVEs?

Reply:
This was a concern of mine as well, since the number of CVEs will fluctuate over time. Currently there are 88 CVEs in the OCP tests and 111 in the GKE tests, so this gives us 4+ pages to work with. It is possible we drift below this number at some point, but I thought it was worthwhile to get the functionality covered.

We could force a lower `perPage` parameter, say from 20 -> 4 to greatly reduce the chances of this being an issue.

Additionally we could mock the results, but that would require separate tests to be useful once the full flow here is implemented.